src/components/ExternalLink/index.tsx

```diff
@@ -15,10 +15,27 @@ const ExternalLink: React.FC<ExternalLinkProps> = ({
   href,
   className
 }) => {
+  const [label, setLabel] = React.useState('');
+
+  useEffect(() => {
+    const links = document.getElementsByTagName('a');
+    const externalButtons = Array.from(links).filter((link) => {
```

Comment:
Nit: These are specifically NOT ExternalLinkButtons and are targetting just ExternalLinks?  If so can we change this variable name to `externalLinks` or something a bit more descriptive of what it is?

Reply:
Ah good catch. Thank yoU! That is absolutely not the right variable name.